docs/extensions/reference/_posts/1970-01-06-CLI.md

```diff
@@ -172,52 +201,46 @@ Also, you can install it on specific a app by providing an app ID. To find out I
 
 ```ShellSession
 $ shoutem install --app {{ site.example.appId }}
-Installing `Restaurnats` extension to `Restaurants` app...
+Installing `Restaurants` extension to `Restaurants` app...
```

Comment:
Restaurants > {{ site.example.extensionName }}?

Reply:
Restaurants has capital R. `site.example.extensionName` is non-capital letter, as is with all extension names.